### ISSUE TITLE
Native integration with Jupyter Notebook

### ISSUE THREAD

**Issue by thexiang** [getpelican/pelican#2616]
Would it be possible to have Jupyter Notebook as native feature?

**Comment by justinmayer** [comment]
Given that this already exists as a plug-in, is there some reason you think this needs to be in Pelican core as opposed to a plug-in?

**Comment by thexiang** [comment]
I recently started using Pelican, the first version I started to use was 4.10, I was able to smoothly configure the theme, domain, and change configurations. but when I try to integrate Jupyter Notebook, most of the resource outside only works for version 3.6, so I believe many beginners like me might be stuck at the same place. and I think Jupyter is becoming a standard format in the Python community, it would be great to have it as a native feature or probably a official documentation, thank you!
